data/wordlists.yml

```diff
@@ -9170,7 +9170,7 @@ DefUserNames:
   categories:
   - usernames
 
-01:
+one:
```

Comment:
This is too generic of a name. Should include "Weakpass" or something that indicates where the wordlist is from.